lib/Config.php

```diff
@@ -229,4 +229,13 @@ public function validateSignalingTicket($userId, $ticket) {
 		return hash_equals($hash, substr($ticket, $lastColon + 1));
 	}
 
+	/**
+	 * Should the room list be hidden for users?
+	 *
+	 * @return bool
+	 */
+	public function hideRoomList() {
+		return $this->config->getSystemValue('spreed.hide_roomlist', false);
```

Comment:
What do you think about naming this `talk-webinary-only-mode` and doing all with one config?
cc @Ivansss

Reply:
Hmm, I would rather keep it separate with different options for room list, participant list, etc.

We could however support an additional property that disables multiple things.

Reply:
The thing is, it makes no sense for normal Talk installations.
And I dont want us to have to test 4 different config combinations on each release, that's why I would prefer a "full" or "restricted" thing, not multiple different levels.